docs/GroupDefs.hpp

```diff
@@ -461,6 +461,11 @@
  * \brief Generic numerical algorithms
  */
 
+/*!
+ * \defgroup NumericalFluxesGroup Numerical Fluxes
+ * \brief The set of available numerical fluxes
```

Comment:
Do we want the system-specific fluxes in here?  If yes, can you go through and add them?  I think there are only two at the moment, one for ScalarWave and one for Burgers.  (The Burgers one is LLF and so can be replaced, but that is probably worth a separate PR.)

Reply:
I thought we wanted all systems to eventually use a system-generic numerical flux. At least NewtonianEuler, RelativisticEuler and GrMhd will use the same fluxes (also Burgers, but let's make sure this PR makes everyone happy first.) Not sure about nonconservative systems in general though.

Reply:
It seems we want all the fluxes in this group (I am also in favor), so can you add them?